app/components/search-facet-daterange/component.js

```diff
@@ -38,6 +37,7 @@ export default Ember.Component.extend({
                 this.send('clear');
             });
         });
+        this.filterUpdated();
```

Comment:
Unfortunately, no. The element isn't there to set the `pickerValue` on during `init`. I will mess around with it though and see if I can get rid of that warning.